src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -748,7 +746,7 @@ class VMBootDiagnostics(ResourceGroupVCRTestBase):
     def __init__(self, test_method):
         super(VMBootDiagnostics, self).__init__(__file__, test_method, resource_group='cli_test_vm_diagnostics')
         self.vm_name = 'myvm'
-        self.storage_name = 'ae94nvfl1k1'
+        self.storage_name = 'clitestbootdiag20170227'
```

Comment:
Static still. Like I mentioned, this is a mitigation, not a true fix at all. The tests already have the cleaning up logic, but if the test engine crashes or for any other reasons the account got left behind, naming it as `clitest` makes a bit easier to spot it and clean up. 
I am under the impression that we want to phase out the recording based test, so I don't plan  the true long-term fix, that is to support random generated account id and normalize the recording before write them into the recording file.